src/views/HomeView.vue

```diff
@@ -290,7 +302,32 @@ p {
 }
 
 #home-card-title {
-  font-size: 2em;
+  font-size: 5rem;
+  color: #fff4d7;
+  font-family: 'Luckiest Guy', serif !important;
+  font-weight: 400;
+  line-height: 5rem;
+  margin: 0 auto 40px auto;
```

Comment:
The margin is a little funky here because the heading "Game Finder" is closer to TheHeader component than it is to the game selection area. We want it to be separated from the top navigation and related to the area with the games, while still giving it room on both sides. So more space above, less below
```suggestion
  margin: 32px auto 16px auto;
```